site/assets/scss/_component-examples.scss

```diff
@@ -439,6 +439,6 @@
 // scss-docs-end sticker-fs-xl
 
 .border-gray-400 {
```

Comment:
You should get rid of this specific Boosted class or rename it. It doesn't exist in Bootstrap, so it's not logical to have something named `.border-gray-400` which doesn't apply `$gray-400`. Either `.border-color-translucent`, either something with context to avoid changing the name each time the value changes, either directly a `.border-*` from the framework which has `--bs-border-color-translucent` as a value.

Reply:
We don't have such a class in Boosted right now, `.border-tertiary` was handling this variable. Should we introduce it for the docs only in a first time ?

Reply:
I think this would be the best approach for now

Reply:
We agreed on changing it to `.border-tertiary`